src/core/Events.js

```diff
@@ -233,14 +233,15 @@ export var Events = {
 		return false;
 	},
 
+	// returns the index (number) or false
 	_listens: function (type, fn, context) {
 		if (!this._events) {
 			return false;
 		}
 
 		var listeners = this._events[type] || [];
 		if (!fn) {
-			return !!listeners.length;
+			return listeners.length > 0 ? 0 : false;

```

Comment:
We can return `true` here like before, it still fits the `!== false` condition where necessary.